packages/terra-alert/docs/README.md

```diff
@@ -0,0 +1,41 @@
+# Terra Alert
+
+The Terra Alert component is a notification banner that can be rendered in your application when there is information that you want to bring to the user's attention.  The Alert component supports a number of built-in notification types that render with pre-defined colors and icons that help the user understand the severity and meaning of the notification. A custom notification type is also supported that allows your application to customize an alert that may not fit into the pre-defined types.
+
+For more details, see [More Information](MoreInformation.md).
```

Comment:
This link doesn't seem to work as expected on the deployment link: https://terra-core-deployed-pr-575.herokuapp.com/static/#/site/alert

We likely need to create a new route for this page and the point to that URL.